docs/src/pages/en/usage/migration.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Migration
+description: Migration
+layout: ../../../layouts/docs.astro
+lang: en
+---
+# Migration
+
+Set up or confirm your development environment by reviewing the [React Native Docs for version 0.7.0](https://reactnative.dev/docs/0.70/environment-setup).
+
+Before starting the project after the FS12 upgrade, please clean your project directory: `git clean-fdx`.
+
+## Troubleshooting
+
+### iOS 
+In addition to cleaning your project directory, we suggest you clear your Xcode Derived Data. To clear it, enter the following in your terminal `rm -rf ~/Library/Developer/Xcode/DerivedData`
+
+### Android
+#### White Text
+> *Note: This will be resolved in the next FS12 release (0.0.11)*
+After upgrading, all text elements in Android that do not have a value for their color will be white when the device is in dark mode. To resolve this, ASSOS patched the template style values XML file so that the application defaulted to light mode, regardless of the device’s current theme (*[See PR](https://github.com/brandingbrand/assos/pull/682)*).
+
+#### Java Vesion
+It's possible OS updates have caused this issue when reported; however, if a Java version issue is encountered while building the Android application, you can resolve it by making sure you set the JAVA_HOME path equal to the one used by Android Studio in your path file.
+
+Open and edit your .zshrc file `code .zshrc`
+Ensure the file contains the following line for Java Version
+
+```
+export JAVA_HOME="/Applications/Android Studio.app/Contents/jre/Contents/Home"
+```
+
+#### Command Line Tools
+If you receive an error message regarding command line tools, such as the following:
+```
+> Cannot run program "/Users/User/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager": error=2, No such file or directory
+```
+
+It’s likely you need to install the Android Command-line Tools within Android Studio.
+
+- Open Android Studio
+- Click on the more actions drop-down in the Welcome Screen.
+- Select SDK Manager
+- View Appearance & Behavior > System Settings > Android SDK
+- Make sure Android SDK Command-line Tools (latest) is selected and installed
+- Click on Apply / Ok
+
+If you note that you already have Command-line Tools installed via Android Studio, you should confirm that your path file correctly lists necessary Android development tools.
+
+- Open and edit your .zshrc file `code .zshrc`
+- Make sure the file contains the following lines for Android Studio.
+
+```
```

Comment:
not sure I agree with including all of this - this all comes straight out of the react native environment setup documentation

Reply:
I think renaming it to `Troubleshooting` and keeping all these details on there would be helpful for current and new users.

I didn't copy and paste this from the react-native documentation. I created these instructions while helping @scbb31994 and @BBryanJ troubleshoot issues we could not solve following the react-native development environment setup docs after an upgrade to FS12.